easybuild/easyblocks/b/bazel.py

```diff
@@ -112,13 +124,47 @@ def configure_step(self):
         # See https://github.com/bazelbuild/bazel/issues/10377
         bazel_args += ' --host_javabase=@local_jdk//:jdk'
 
+        # Link C++ libs statically, see https://github.com/bazelbuild/bazel/issues/4137
+        env.setvar('BAZEL_LINKOPTS', '-static-libstdc++:-static-libgcc')
+        env.setvar('BAZEL_LINKLIBS', '-l%:libstdc++.a')
+
         env.setvar('EXTRA_BAZEL_ARGS', bazel_args)
+        env.setvar('EMBED_LABEL', self.version)
+        env.setvar('VERBOSE', 'yes')
 
     def build_step(self):
         """Custom build procedure for Bazel."""
-        cmd = '%s ./compile.sh' % self.cfg['prebuildopts']
+        cmd = ' '.join([
+            "TMPDIR='%s'" % self.bazel_tmp_dir,  # The initial bootstrap of bazel is done in TMPDIR
+            self.cfg['prebuildopts'],
```

Comment:
Either you want to change the order here, or you should use `export TMPDIR='%s' && `.

`prebuildopts` is often used to "glue" commands before the build command, and then `TMPDIR` won't be set for the `./compile.sh` command...